src/kademlia/node_id.cpp

```diff
@@ -198,10 +198,10 @@ node_id generate_id(address const& ip)
 	return generate_id_impl(ip, random(0xffffffff));
 }
 
-bool matching_prefix(node_entry const& n, int mask, int prefix, int bucket_index)
+bool matching_prefix(node_entry const& n, int mask, int prefix, int offset)
```

Comment:
I don't see how I could test `matching_prefix`, the special case is handled by the caller.